main.py

```diff
@@ -1,27 +1,24 @@
-from foodfuncs import *
-import sqlite3
+import foodfuncs as ff
+import utils as u
 
 
-ifrec = input("Would you like to add another favorite recipe? Please type \'y\' or \'n\':  ")
-if ifrec == 'y':
-    recname = input("Please enter the name of your recipe:  ")
-    instruct = input("Please type the instructions for your recipe:  ")
-    ingdict = {}
-    ingr = ''
-    while ingr != 'x':
-        ingr = input("Enter the ingredient name. If there are no more ingredients, enter 'x' :  ") 
-        amt = input("Enter the amount of the ingredient as used in your recipe:  ")
-        if ingr != 'x' and ingr != 'X':
-            ingdict[ingr.lower()] = amt.lower()
-    print(f"Are these ingredients and amounts correct for your {recname} recipe?")
-    for x in ingdict:
-        print(x, ":", ingdict[x])
-    add_new_recipe(recname, instruct, ingdict)
-elif ifrec == 'n':
-    pick = input("Would you like a random recipe?")
+ifrec = input("Would you like to add another favorite recipe? Please type \'yes\' or \'no\':  ")
```

Comment:
I think it would be best practice to put this logic in a function, and then call on it with `if __name__ == "__main__"`. That will allow you to continue having the functionality of this code being run when calling on the file.

Reply:
To clarify, do you think it would be better to make this a function and move it into foodfuncs or util, then have main.py contain only `if __name__ == "__main__"` or to leave the function in main.py?